Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN apt-get update -qq && \
                        openssl \
                        psmisc \
                        dsniff \
+                       mtr-tiny \
```

Comment:
AFAIK there's also an "mtr" package. If that's correct, do you know what the difference to this tiny variant would be?

Reply:
It's the curses-only version (no gui). I put more context in the internally linked issue with details on how I tested that it is used the same way: https://digitalocean.slack.com/archives/C4HREK3T6/p1676078759996069

Reply:
Makes sense.